src/ds/ptrwrap.h

```diff
@@ -213,7 +269,7 @@ namespace snmalloc
    * several chunks) to be the allocation.
    */
   template<typename T>
-  inline SNMALLOC_FAST_PATH CapPtr<T, CBAllocE>
+  SNMALLOC_FAST_PATH CapPtr<T, CBAllocE>
```

Comment:
Did you mean to move the inline after SNMALLOC_FAST_PATH?

Reply:
Oh it's a mess and/or I don't know what I'm doing.  MSVC objects to `__forceinline inline`, which rules out `SNMALLOC_FAST_PATH inline` given the current definitions, but GCC sometimes balks at `__attribute__((always_inline))` without `inline`.  See 8fc0dcc and ideally save me from myself and C++.

Reply:
I don't like this.  `inline` is really about ODR linkage, and the macro is about adding compiler attributes for assuming this is a very common path.

Reply:
Yea, I don't like it either.  Open to suggestions given the constraints. :(

Reply:
Hm, OK, apparently this is all just me being overzealous with `SNMALLOC_FAST_PATH`.  Lemme rip that out and see if things get better. :)

Reply:
Yea, that looks better!